### PR TITLE
[rake] Update the value of APIcast master access token

### PR DESCRIPTION
**Pull request by hallelujah**
This task allow restoring the APIcast master access token

### CODE REVIEW

File: lib/tasks/restore/access_token.rake [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+namespace :restore do
```
Comment (guicassolato) [inline lib/tasks/restore/access_token.rake:3]: Are we planning on having/moving other tasks under `lib/tasks/restore`?

Comment (hallelujah) [reply lib/tasks/restore/access_token.rake:3]: Well I could just create without namespace but I have no idea

Comment (hallelujah) [reply lib/tasks/restore/access_token.rake:3]: it could be `access_tokens:master:update` wdyt?

**Comment by hallelujah** [comment]
Closing in favor of #2462

**Pull request closed without merging**